Formula/dtkcore@2.rb

```diff
@@ -0,0 +1,73 @@
+class DtkcoreAT2 < Formula
+  desc "Deepin Tool Kit Core Module"
+  homepage "https://www.deepin.org"
+  url "http://packages.deepin.com/deepin/pool/main/d/dtkcore/dtkcore_2.0.16.1.orig.tar.xz"
+  sha256 "79b19a9b01f8a53d2794afa3073d3303b54d6e07da94cc1ef20eb32ea0c3f465"
+
+  depends_on "pkg-config" => :build
+  depends_on "qt"
+
+  patch :DATA
```

Comment:
New formula shouldn't have patches, please request upstream to fix it and make a formula with the new version.

Reply:
dtkcore@2 has been archived and we are developing a new version, so I can only compile by patch.

Reply:
Then please wait until the new version is released.